src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -83,9 +82,14 @@
        [:f> composer.view/composer
         {:insets                            insets
          :scroll-to-bottom-fn               list.view/scroll-to-bottom
-         :show-floating-scroll-down-button? list.view/show-floating-scroll-down-button?}]
+         :show-floating-scroll-down-button? show-floating-scroll-down-button?}]
        [contact-requests.bottom-drawer/view chat-id contact-request-state group-chat])]))
 
 (defn chat
   []
-  [:f> f-chat])
+  (let [reagent-atoms
```

Comment:
nitpick: rename to inner-state-atoms, reagent-atoms is too genetic imo.

Reply:
done, thank you